about.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/about.css">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Men International Conference 2023</title>
+</head>
+<body>
+ <header>
+    <nav>
+        <div class="top-nav">
+            <div class="top-nav__container">
+                <img src="asset/images/facebook.svg" alt="facebook-logo" class="icon">
+            <img src="asset/images/twitter.svg" alt="facebook-logo" class="icon">
+            <ul class="top-nav__menu">
+                <li class="top-nav__menu-item"><a href="#">English</a></li>
+                <li class="top-nav__menu-item"><a href="#">My Page</a></li>
+                <li class="top-nav__menu-item"><a href="#">Logout</a></li>
+            </ul>
+            </div>
+        </div>
+        <div class="lower-nav">
+           <div class="lower-nav__center hide" id="menu">
+            <img src="asset/images/logo.png" alt="logo" class="lower-nav__logo">
+            <ul class="logo__menu-list">
+                <li class="menu-list__item"><a href="index.html">Home</a></li>
+                <li class="menu-list__item"><a href="about.html">About</a></li>
+                <li class="menu-list__item"><a href="#speakers">Program</a></li>
+                <li class="menu-list__item"><a href="#join">Join</a></li>
+                <li class="menu-list__item"><a href="#partners">Partners</a></li>
+                <li class="menu-list__item cc-campaign"><a href="#">CC Campaign</a></li>
+            </ul>
+           </div>
+            </div>
+            <div class="lower-nav__hamburger" id="hambuger">
+                <img src="asset/images/hamburgerImg.svg" alt="hambuger" class="open ham-icon" id="openHam">
+                <img src="asset/images/close.png" alt="hambuger" class="close hide ham-icon" id="closeHam">
+            </div>
+    </nav>
+    <div class="about__hero" id="about__hero">
+        <h3 class="about__hero-title">"Hello! Gentle Men"</h3>
+        <h1 class="about__hero-heading">INTERNATIONAL MEN EMPOWERMENT SCHEME 2023</h1>
+        <p class="about__hero-body">The International Women Empowerment Scheme brings together a community of women, If you have benefitted from this and would like to volunteer to help other Men, please consider joining us to build a stronger community of successful Men. </p>
+        <p class="about__hero-contact">Please contact us via email for further questions about IWES conference 2023!</p>
+        <p class="about__hero-email"><a href="#">womenconference@IWES.org</a></p>
+    </div>
```

Comment:
Kindly update the hero section of your about page on **both** mobile and desktop to match the behance design better🙏

Your design
<img width="948" alt="image" src="https://user-images.githubusercontent.com/37116322/232931643-76863d0e-4d20-4676-94ee-3bc488ee508b.png">

Behance design
<img width="554" alt="image" src="https://user-images.githubusercontent.com/37116322/232931934-875a53db-77cc-4dcb-b37d-d102343001e5.png">

A few things to note:
- Please add more bottom padding and increase the spacing between the elements
- Please make sure the text in this section is centered